docs/reference-implementation/Getting-Started.md

```diff
@@ -161,7 +161,7 @@ az pipelines create --name "Azure.AlwaysOn E2E Release" --description "Azure.Alw
 
 All pipelines require an Azure DevOps service connection to access the target Azure Subscription where the resources are deployed. These service connections use Service Principals to access Azure which can be configured automatically, when proper access is given, or manually in Azure DevOps by providing a pre-created Azure Service Principal with the required permissions.
 
-We need to create an AAD Service Principal with **Subscription-level Owner permissions**. We need owner permission as the pipeline will need to create various role assignments.
+> **Important:** You need to create an AAD Service Principal with **Subscription-level Owner permissions**. We need owner permission on the whole subscription as the pipeline will need to create various role assignments.
```

Comment:
```suggestion
> **Important!** The AAD Service Principal needs **subscription-level owner permissions** as the pipeline will create various role assignments.
```

Reply:
Ok, let's increase the urgency a bit :D

Reply:
**Important!** for consistency - this is the notation that's used in 3 other cases in this document.

And the second part for easier readability - its more likely that someone reads a single sentence than 2.. up to you. Just an idea.

Reply:
I like it! More concise and clear language always wins in my view.